tests/test_backends.py

```diff
@@ -117,8 +117,8 @@ def resultIterator(
         not_done = True
         request.pageSize = pageSize
         while not_done:
-            response = searchMethod(request)
-            self.assertEqual(type(response), ResponseClass)
+            responseStr = searchMethod(request.toJSON())
+            response = ResponseClass.fromJSON(responseStr)
             l = getattr(response, listMember)
```

Comment:
Definitely. Again, the required method is being added in #64, so I've put in a TODO.